File: CompSci 1/Unit 16, Iteration - for/Vertical String/verticalString.java
import java.util.Scanner;

public class verticalString {
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);

        System.out.print("Enter a string: ");

        String word = input.nextLine();

        for(int i = 0; i < word.length(); i++)
        {
            System.out.println(word.substring(i, i+1));
        }
    }
}

File: CompSci 1/Unit 16, Iteration - for/Vertical String/verticalString.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val input = Scanner(System.`in`)

    print("Enter a string: ")

    val word = input.nextLine()

    for (i in 0 until word.length) {
        println(word.substring(i, i + 1))
    }
}
